dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -470,6 +470,15 @@ private void resolve(Object installObject) {
         }
     }
 
+    public boolean compareIfContains(Collection<String> item1, String item2) {
```

Comment:
- Call the method something like containsIgnoreCase, since current name does not tell you much.
- Avoid parameter names "item1" and "item2", since the parameter types are not the same.